cluster/ci/config/prow/config.yaml

```diff
@@ -319,6 +319,15 @@ presubmits:
     rerun_command: "/test containerized"
     trigger: "((?m)^/test( all| containerized),?(\\s+|$))"
 
+  - name: test_pull_request_openshift_ansible_extended_conformance_install_crio
+    agent: jenkins
+    labels:
+      master: ci.openshift.redhat.com
+    always_run: true
```

Comment:
@sdodson non-blocking but reporting often confuses devs -- maybe make this not report but get the people who are invested in it to look at the job?

Reply:
Should we do that if the intention is to make this blocking?

Reply:
Yeah, it will keep noise down while the job isn't blocking -- realistically if it's not blocking only the devs that own the job will end up caring until it blocks

Reply:
Yeah, also note that in the near future every job that always runs will be considered blocking.